third_party/docfx/templates/devsite/ManagedReference.common.js

```diff
@@ -16,13 +16,14 @@ exports.transform = function (model) {
   }
 
   if (model.type) {
+    var typePropertyName = getTypePropertyName(model.type)
+    model[typePropertyName] = true;

```

Comment:
What happens when we encounter a type that's not specified below?

Reply:
This would set ```model[typePropretyName]``` for types not covered in the switch statement, or set ```model[undefined] = true;``` in the case it's not a valid type. I think we should keep the behavior the same as before, so I'll revert this to have ```model[typePropertyName]``` be set on a case by case basis.